expression/integration_serial_test.go

```diff
@@ -3813,12 +3813,12 @@ func TestSetVariables(t *testing.T) {
 	require.Error(t, err)
 	require.Error(t, err, variable.ErrWrongTypeForVar.GenWithStackByArgs("max_prepared_stmt_count").Error())
 
-	tk.MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("1"))
-	require.True(t, variable.EnableConcurrentDDL.Load())
-	tk.MustExec("set @@global.tidb_enable_concurrent_ddl=0")
-	tk.MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("0"))
-	require.False(t, variable.EnableConcurrentDDL.Load())
-	testkit.NewTestKit(t, store).MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("0"))
+	//tk.MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("1"))
+	//require.True(t, variable.EnableConcurrentDDL.Load())
+	//tk.MustExec("set @@global.tidb_enable_concurrent_ddl=0")
+	//tk.MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("0"))
+	//require.False(t, variable.EnableConcurrentDDL.Load())
+	//testkit.NewTestKit(t, store).MustQuery("select @@global.tidb_enable_concurrent_ddl").Check(testkit.Rows("0"))
```

Comment:
Please uncomment it or remove it with reasons.

Reply:
we will add this test back

Reply:
It has been changed in the master. I think it can uncomment it.